.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name: Release server container image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "version (no `v` prefix)"
+        required: true
+        default: "1.0.0"
+
+env:
+  IMG_REGISTRY_HOST: quay.io
+  IMG_REGISTRY_ORG: kuadrant
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: server-v${{ github.event.inputs.version }}
```

Comment:
This is a non-neglectable change from the `build-image.yaml`…
The rest is configuration changes, so we'd need to add logic to configure one way or the other depending on the way the script was triggered… Wonder if logic is more desirable than plain declarative workflows… but I'll let you be the judge. At this stage I think it's pretty even one way or the other

Reply:
I wonder if we can avoid both. Let me try to illustrate how I see it.

The only manual action should be tagging the repo. Then tagging itself should trigger other workflows like docker image build and publish crate to crate.io. So, basically, fot the docker images we already have it in `build-image.yaml`

```yaml
on: 
  push:
    branches: [*]
    tags: [*]
```

The "Release crate" workflow should also be triggered by tagging, not by manually triggered workflow. We may want to define better the triggering to only publish a crate for some kind of tags, like `vX.Y.Z` or whatever. Then allow adding other tags will not publish crates.